@xen-orchestra/web-core/lib/components/ui/UiTreeItem.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <li class="ui-tree-item">
+    <slot />
+    <slot v-if="isExpanded" name="sublist" />
+  </li>
+</template>
+
+<script lang="ts" setup>
+import { IK_LIST_ITEM_EXPANDED, IK_LIST_ITEM_HAS_CHILDREN, IK_LIST_ITEM_TOGGLE } from '@core/utils/injection-keys.util'
```

Comment:
Would be better if Injection Keys names match the component name.

```suggestion
import { IK_TREE_ITEM_EXPANDED, IK_TREE_ITEM_HAS_CHILDREN, IK_TREE_ITEM_TOGGLE } from '@core/utils/injection-keys.util'
```